### DISCUSSION TITLE
Release Candidates for 1.9

### DISCUSSION THREAD

**Discussion by lecaros** [fluent/fluent-bit#4954] [Announcements]
Hello everyone!
Starting with 1.9, we're tagging and shipping RCs builds of Fluent Bit. We'd like to know if you're testing them and how much you're loving them already. Also, let us know of any issues you may find while playing with them.
The last one we've shipped is [1.9.0-rc2](https://github.com/fluent/fluent-bit/releases/tag/v1.9.0-rc2). If you're looking for the images, you can find them [here](https://github.com/fluent/fluent-bit/pkgs/container/fluent-bit%2Fstaging).

What's new?
- YAML support for config.
- Kafka input plugin
- Nginx metrics
- OpenSearch output plugin
- Apache Skywalking
- Windows metrics
- Default workers
- BigQuery output plugin
- Opentelemetry output plugin
- Apache arrow support
- and various fixes!

We'd love your feedback for these releases!

**Comment by lecaros** [comment]
Hello, everyone.
We've just released [1.9.0-rc3](https://github.com/fluent/fluent-bit/releases/tag/v1.9.0-rc3)!

**Comment by patrick-stephens** [comment]
[1.9.0-rc4](https://github.com/fluent/fluent-bit/releases/tag/v1.9.0-rc4) is out now as well.

**Comment by patrick-stephens** [comment]
RC packages are here: https://github.com/fluent/fluent-bit/actions/workflows/staging-build.yaml
Containers are documented here: https://github.com/fluent/fluent-bit/tree/master/dockerfiles#ghcrio-topology
Staging currently contains the RC containers: [ghcr.io/fluent/fluent-bit/staging](http://ghcr.io/fluent/fluent-bit/staging)

**Comment by patrick-stephens** [comment]
1.9.0 is released: https://fluentbit.io/announcements/v1.9.0/
